parity/main.rs

```diff
@@ -269,7 +269,7 @@ fn setup_rpc_server(
 			}
 		}
 	}
-	Some(server.start_http(url, cors_domain, 1))
```

Comment:
Just quick and dirty - whole code is going to be refactored. Static files are now served by the same server and some binary files really slows the loading down.